main.tf

```diff
@@ -6,10 +6,10 @@ resource "random_integer" "this" {
 locals {
   account_name          = coalesce(var.account_name, "azure-openai-${random_integer.this.result}")
   custom_subdomain_name = coalesce(var.custom_subdomain_name, "azure-openai-${random_integer.this.result}")
-  tags = var.default_tags_enabled ? {
+  tags = merge(var.default_tags_enabled ? {
```

Comment:
@soferreira do we want to give the possibility to add additional tags also when the default tags are enabled ? Or this version of the logic works ?

@JinLee794 feedback is welcome

Reply:
Ah wait, I misunderstood the code.

I understand now correctly that `var.tags` is always merged in the tags, regarless of the value of `var.default_tags_enabled`. 

Looks good to me !

Reply:
Looks to me!

Reply:
Looks good to me!